src/utils.cairo

```diff
@@ -237,3 +237,20 @@ pub fn from_base16(hexs: ByteArray) -> @ByteArray {
 
     @result
 }
+
+pub fn from_string_to_hexa(hexs: ByteArray) -> @ByteArray {
+    let mut result: ByteArray = Default::default();
+    let mut i = 0;
+    let len = hexs.len();
+    assert_eq!(len % 2, 0);
```

Comment:
just a random info  : `assert_eq!` macro is very gas consuming,  we generally use it only on tests. But yeah this is a helper for testing purpose only, so its fine. but its probably a bit bette to use `assert` function